src/app/explore/explore.component.ts

```diff
@@ -124,6 +130,16 @@ export class ExploreComponent implements OnDestroy, OnInit {
   }
 
   onScroll() {
+    if (this.scroller.getScrollPosition()[1] < this.smallestSignificantScrollPx) {
+      this.blurredSinceLastMajorScroll = false;
+      return;
```

Comment:
This early return may stop some `more()` calls, but since we do the equivalent on meta-campaign already I can't see that causing major problems.